godtools/FlowControllers/BaseFlowController.swift

```diff
@@ -42,4 +43,55 @@ class BaseFlowController: NSObject, UINavigationControllerDelegate {
         navigationController.delegate = self
     }
     
+    // Notifications
+    
+    func defineObservers() {
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(displayMenu),
+                                               name: .displayMenuNotification,
+                                               object: nil)
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(dismissMenu),
+                                               name: .dismissMenuNotification,
+                                               object: nil)
+    }
+    
+    func displayMenu() {
+        let menuViewController = MenuViewController(nibName: "MenuViewController", bundle: nil)
+        let navigationController = self.currentViewController?.navigationController
+        let src = self.currentViewController
+        let dst = menuViewController
+        let srcViewWidth = src?.view.frame.size.width
+        
+        src?.view.superview?.insertSubview(dst.view, aboveSubview: (src?.view)!)
```

Comment:
If you unwrap your optionals along the way, then you don't have to unwrap the result. The same dangers of unwrapping apply, but the code is cleaner in my opinion. So you'd have here:
````swift
src?.view.superview?.insertSubview(dst.view, aboveSubview: src!.view)
````
(note the first `src` reference actually makes this statement safe b/c if it's nil the second one that we force unwrap would never get evaluated)